tool/gravity/cli/rpcagent.go

```diff
@@ -374,7 +420,7 @@ func getGravityPackage() loc.Locator {
 type deployAgentsRequest struct {
 	clusterEnv   *localenv.ClusterEnvironment
 	clusterState storage.ClusterState
-	clusterName  string
+	cluster      ops.Site
```

Comment:
Do we still need a separate `clusterState` field in the request? Looks like `ops.Site` has a `ClusterState` field.

Reply:
Yeah, we do. I thought the same originally, but some operations calculate cluster state a bit differently: https://github.com/gravitational/gravity/blob/version/5.5.x/tool/gravity/cli/update.go#L115-L117.